lib/plugins/emit/index.test.js

```diff
@@ -177,10 +178,10 @@ describe('Emit', () => {
             data: emit.data,
           })
         ).to.equal(true);
-        expect(logStub.getCall(0).args[0]).to.equal(
-          // eslint-disable-next-line max-len
-          ' Serverless       Emitted the event userCreated as datatype application/json:\n                    {\n                      "key": "value"\n                    }'
-        );
+        expect(logStub.getCall(0).args[0]).to.equal([
+          ' Serverless       Emitted the event userCreated as datatype application/json:',
+          '                    {\n                      "key": "value"\n                    }',
```

Comment:
This is the string as seen on Windows and what gets the test to pass. I do not know if the _intent_ of the code is to write literal `\n` and so this is the correct test, or if the code should write `os.SEP` which would make it a bug in emit/index.js, caused by this: https://github.com/serverless/serverless/blob/45c07f519f960bc7cd65acf1392791476b51bcb7/lib/plugins/emit/index.js#L20-L21